README.md

```diff
@@ -887,6 +887,7 @@ Below is the complete list of available options that can be used to customize yo
 | `GITLAB_MATTERMOST_ENABLED` | Enable/Disable GitLab Mattermost for *Add Mattermost button*. Defaults to `false`. |
 | `GITLAB_MATTERMOST_URL` | Sets Mattermost URL. Defaults to `https://mattermost.example.com`. |
 | `GITLAB_BACKUP_SCHEDULE` | Setup cron job to automatic backups. Possible values `disable`, `daily`, `weekly` or `monthly`. Disabled by default |
+| `GITLAB_BACKUP_STRATEGY` | Set GitLab backup strategy. No defaults, which means it uses stream strategy [See](https://docs.gitlab.com/ee/raketasks/backup_restore.html#backup-strategy-option) |
```

Comment:
Here should be the link to the documentation for the community edition and not enterprise edition replace this with this https://docs.gitlab.com/ce/raketasks/backup_restore.html#backup-strategy-option